server/server.go

```diff
@@ -556,6 +560,23 @@ func (s *server) findAndSetDefaultService() {
 	}
 }
 
+func (s *server) unknownMethodHandler() error {
```

Comment:
Not a name good enough for this. Better be `registerUnknownMethodHandler`

Reply:
okay